test/verify/check-journal

```diff
@@ -455,54 +464,136 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
+            b.set_val(sel, test_user)
+
+            sel = "#cockpit_modal_dialog .modal-footer button:contains('Send')"
+            b.click(sel)
+
+        sel = "#cockpit_modal_dialog .modal-content p:contains('Password')"
+        b.wait_not_present(sel)
+
+        sel = "#cockpit_modal_dialog .modal-footer button:contains('No')"
+        b.click(sel)
+
+        sel = "table.reporting-table tr:nth-child(2) a[href='https://bugzilla.example.com/show_bug.cgi?id=123456']"
         b.wait_present(sel)
 
-        self.allow_journal_messages('.*This problem has already been reported.')
-        self.allow_journal_messages('.*http://localhost:12345/reports/42/')
-        self.allow_journal_messages('.*https://bugzilla.example.com/show_bug.cgi\?id=123456')
+    @skipImage("ABRT does not work on i386", "fedora-i386")
+    @skipImage("ABRT not available", "debian-stable", "debian-testing", "ubuntu-stable",
+               "ubuntu-1804", "fedora-coreos", "rhel-8-1", "rhel-8-1-distropkg", "rhel-8-2")
+    def testAbrtReportCancel(self):
+        self.allow_core_dumps = True
+
+        b = self.browser
+        m = self.machine
+
+        m.upload(["verify/files/mock-bugzilla-server.py"], "/tmp/")
+        m.execute("setsid /tmp/mock-bugzilla-server.py >/tmp/mock-bugzilla.log 2>&1 &")
+        m.execute("echo 'BugzillaURL=http://localhost:8080' >> /etc/libreport/plugins/bugzilla.conf")
 
-        sel = "#journal-entry-fields .nav .btn-primary:contains('Report')"
+        # start mock FAF server
+        m.upload(["verify/files/mock-faf-server.py"], "/tmp/")
+        m.execute("setsid /tmp/mock-faf-server.py >/tmp/mock-faf.log 2>&1 &")
+        m.execute("echo 'URL=http://localhost:12345' >> /etc/libreport/plugins/ureport.conf")
+
+        self.crash()
+
+        self.login_and_go("/system/logs")
+
+        sel = "#journal-box .cockpit-logline .cockpit-log-message:contains('crashed in __nanosleep()')"
         b.click(sel)
 
-        # jQuery magic on this page updates the whole frame for changing to "Reported" state
-        b.expect_load_frame("cockpit1:localhost/system/logs")
-        sel = "#journal-entry-fields .nav .problem-btn:contains('Reported')"
+        # Something long-running to not pop up an unexpected dialog.
+        m.execute("sed -i 's|abrt-action-perform-ccpp-analysis|echo Cancel me; sleep 5m|' /etc/libreport/events.d/ccpp_event.conf")
+
+        sel = "table.reporting-table tr:nth-child(2) button:contains('Report')"
+        b.click(sel)
+
+        sel = "table.reporting-table tr:nth-child(2) td:contains('Cancel me')"
         b.wait_visible(sel)
-        self.assertIn("/reports/bthash/123deadbeef", b.attr(sel, 'href'))
 
+        sel = "table.reporting-table tr:nth-child(2) button:contains('Cancel')"
+        b.click(sel)
+
+        sel = "table.reporting-table tr:nth-child(2) td:contains('Reporting was canceled')"
+        b.wait_present(sel)
+
+    def testAbrtReportNoReportd(self):
```

Comment:
This test case needs to be skipped on OSes without ABRT, like here: https://logs-https-cockpit.apps.ci.centos.org/logs/pull-11150-20191118-131111-9511f328-cockpit-project-cockpit--debian-testing/log.html#253-2

Reply:
Argh, completely forgot about it.